config/main.py

```diff
@@ -12,17 +12,20 @@
 from swsssdk import ConfigDBConnector
 from natsort import natsorted
 from minigraph import parse_device_desc_xml
+from portconfig import get_port_config
```

Comment:
Why add this import if you are not using it?

Reply:
removed in the new commit